src/DropdownFilter.xml

```diff
@@ -0,0 +1,76 @@
+        </property>
+        <property key="placeholder" type="translatableString">
+            <caption>Placeholder</caption>
+            <category>General</category>
+            <description>Used as filter placeholder when no default filter exists</description>
+            <translations>
+                <translation lang="en_US">Select...</translation>
+                <translation lang="nl_NL">Kiezen...</translation>
+            </translations>
+        </property>
+        <property key="filters" type="object" isList="true">
+            <caption>Filters</caption>
+            <category>Filters</category>
+            <description/>
+            <properties>
+                <property key="caption" type="translatableString">
+                    <caption>Caption</caption>
+                    <category>General</category>
+                    <description/>
+                </property>
+                <property key="filterBy" type="enumeration" defaultValue="attribute">
+                    <caption>Filter on</caption>
+                    <category>General</category>
+                    <description/>
+                    <enumerationValues>
+                        <enumerationValue key="attribute">Attribute</enumerationValue>
+                        <enumerationValue key="XPath">XPath</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="attribute" type="attribute" required="false" entityProperty="../entity">
+                    <caption>Attribute</caption>
+                    <category>General</category>
+                    <description>The attribute on which the filter method is applied to</description>
+                    <attributeTypes>
+                        <attributeType name="String"/>
+                        <attributeType name="Integer"/>
+                        <attributeType name="Decimal"/>
+                        <attributeType name="Boolean"/>
+                    </attributeTypes>
+                </property>
+                <property key="attributeValue" type="string" required="false">
+                    <caption>Attribute value</caption>
+                    <category>General</category>
+                    <description/>
+                </property>
+                <property key="constraint" type="entityConstraint" required="false" entityProperty="../entity">
+                    <caption>XPath constraint</caption>
+                    <category>General</category>
+                    <description>Constraint on the data from the data entity (used when comparison is XPath)</description>
+                </property>
+                <property key="isDefault" type="boolean" defaultValue="false">
+                    <caption>Default selected filter</caption>
+                    <category>General</category>
+                    <description>Only one filter can be set as default</description>
```

Comment:
`Only one filter can be set as default. If no "Default selected filter" is set, the first filter is selected initially, unless "Enable empty filter" is enabled`

Reply:
Added